pkg/controllermanager/controller/certificatesigningrequest/csr_autoapprove_suite_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Can you rename this file to match the package name?
I.e., `certificatesigningrequest_suite_test.go`